source/tutorials/openmm.rst

```diff
@@ -17,4 +23,128 @@ folder of the GitHub repository. It contains:
   NanoVer simulation using a python client.
 * `openmm_neuraminidase`: A notebook that demonstrates how to set up an interactive
   OpenMM simulation from scratch with NanoVer and change the visualisations of
-  different atom selections for a protein-ligand system.
\ No newline at end of file
+  different atom selections for a protein-ligand system.
+
+XML Files
+==============================================
+
+The XML format for serializing OpenMM simulations consists of a root element ``<OpenMMSimulation>`` containing three main components:
+
+1. **Starting Structure**: Enclosed in either ``<pdbx>`` or ``<pdb>`` tags
+2. **OpenMM Serialized System**: Enclosed in the ``<System>`` tag
+3. **OpenMM Serialized Integrator**: Enclosed in the ``<Integrator>`` tag
+
+An optional fourth component, the OpenMM serialized state, may also be included.
+The format allows for efficient storage and exchange of OpenMM simulation data, including the starting structure, system configuration, and integrator settings.
+
+XML Structure
```

Comment:
Each word starts with an upper case in the sub headings on this page, which is inconsistent with the rest of the docs. Would be good to change these so only the first letter starts with an upper case.